addons/lunch/models/lunch.py

```diff
@@ -0,0 +1,296 @@
+# -*- coding: utf-8 -*-
+
+import datetime
+from openerp import models, fields, api, _
+
+class lunch_order(models.Model):
+    """
+    lunch order (contains one or more lunch order line(s))
```

Comment:
NotImportant: This docstring does not add anything to the model understanding. So either we improve it if there are things to say about the model, either we remove it.